x-pack/plugins/apm/server/lib/services/get_service_instances/detailed_statistics.ts

```diff
@@ -123,9 +123,7 @@ export async function getServiceInstancesDetailedStatisticsPeriods({
         previousPeriodPromise,
       ]);
 
-      const firtCurrentPeriod = currentPeriod.length
-        ? currentPeriod[0]
-        : undefined;
+      const firstCurrentPeriod = currentPeriod?.[0];
```

Comment:
Thanks for refactoring it 👍🏻

Reply:
No problem :)
Btw. did you mean to delete this comment?

![image](https://user-images.githubusercontent.com/209966/128721077-50a264a4-9711-4457-ba4a-57ada29e1909.png)

Reply:
No problem :)